### ISSUE TITLE
JAWS could narrate "checked" stated for radiogroup in toolbar

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/VFO-standards-support#557]
### Summary

Example:
1. Go to[ https://www.w3.org/TR/wai-aria-practices-1.1/examples/toolbar/toolbar.html]( https://www.w3.org/TR/wai-aria-practices-1.1/examples/toolbar/toolbar.html)
2. Navigate into the toolbar 
3. Use right arrow key until you reach radio button with name "Text Align Left" 
4. Use right arrow key to navigate to next radio button
5. press Enter/Space

### Actual result
Radio item/button is selected. 
JAWS is silent.

### Expected result
JAWS could narrated information that item was selected/checked. 
Similar way as JAWS narrated that radio button is checked here:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/radio/radio-1/radio-1.html

### Example
[ https://www.w3.org/TR/wai-aria-practices-1.1/examples/toolbar/toolbar.html]( https://www.w3.org/TR/wai-aria-practices-1.1/examples/toolbar/toolbar.html)

#### JAWS version and build number
2021.2103.174
 
#### Operating System and version
Win10 enterprise
10.0.19043

#### Browser and version:
Chrome
Version 92.0.4515.107 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
The same applies here as for: https://github.com/FreedomScientific/VFO-standards-support/issues/556#issuecomment-886570223. JAWS does not output the status change after activating a radio button with Enter or Space bar, because radio buttons cannot normally be activated with the Enter or Space at all (when JAWS is in forms mode).

The output is correct when activated with the Enter or Space when the JAWS virtual cursor is on.

I.e. this is also rather a bug of the toolbar example of WAI-ARIA.

JAWS could, of course, output any state change to be open to non-standard radio buttons.

**Comment by StephenBeckettTPGi** [comment]
For reference purposes: https://www.w3.org/TR/wai-aria-practices-1.2/#for-radio-group-contained-in-a-toolbar

**Comment by kolaps33** [comment]
tested today with JAWS 2023.2212.23
and I can hear after user do selection, JAWS narrates the state change to checked.
Enclosing JAWS narration screenshot:

![image](https://user-images.githubusercontent.com/22620150/213193301-2f2c7c2e-0913-4dda-b501-f7b3c804c4b9.png)

**Comment by kolaps33** [comment]
tested today with JAWS 2023.2212.23
and I can hear after user do selection, JAWS narrates the state change to checked.
Enclosing JAWS narration screenshot:
![image](https://user-images.githubusercontent.com/22620150/213193846-258169d6-7c5e-45fe-a6b9-dedb06e558f0.png)
